paddle/fluid/platform/port.h

```diff
@@ -24,43 +24,42 @@
 #include "glog/logging.h"
 
 #if !defined(_WIN32)
-#define UNUSED __attribute__((unused))
-#include <dlfcn.h>     //  dladdr
-#include <execinfo.h>  // backtrace
-#include <sys/stat.h>
-#include <algorithm>  // std::accumulate
+    #define UNUSED __attribute__((unused))
```

Comment:
we need to clean up the port header. There is a lot of other UNUSED macro in other files.

Reply:
ok